custom-plugin-rules/markdown-validator/rule-validate-markdown.js

```diff
@@ -0,0 +1,77 @@
+module.exports = ValidateMarkdown
```

Comment:
Please move the exports to the end of the file. And run prettier.

Reply:
Done!